hc/api/migrations/0027_checkreport.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```

Comment:
Why are there `migration` files in your PR? Have the models changed?

Reply:
The migrations file had been ignored earlier thinking they would cause an error once we pull and use different databases on our local environments.

However, we re-included them after learning that they cause no harm. My review onTuesday on this very PR has a much more detailed explanation

Reply:
If the models have not changed, remove the migration files